docs:update links and revise docs instructions (#101)
appliedzkp/semaphore: docs/README.md

    <img alt="Repository top language" src="https://img.shields.io/github/languages/top/akinovak/semaphore-spec?style=flat-square">
</p>

---

This directory contains the Semaphore documentation published at [semaphore.appliedzkp.org/docs](https://semaphore.appliedzkp.org/docs).

## Build and run Semaphore docs

> Semaphore documentation is written in Markdown syntax.
> [semaphore.appliedzkp.org/docs](https://semaphore.appliedzkp.org/docs) uses the [Docusaurus](https://docusaurus.io/) site generator and JavaScript tooling to run, build, and deploy the web pages.

Author: ```suggestion
Semaphore documentation uses Markdown syntax.
[semaphore.appliedzkp.org/docs](https://semaphore.appliedzkp.org/docs) uses the [Docusaurus](https://docusaurus.io/) site generator and JavaScript tooling to build and deploy styles, scripts, and HTML.
```
